.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```
@@ -0,0 +1,52 @@
+name: Feature Request
+description: A suggestion for a new feature
+labels: [enhancement]
+body:
+- type: markdown
+  attributes:
+    value: "## Thank you for contributing to Swift-DocC!"
+- type: markdown
+  attributes:
+    value: |
+      Before you submit your issue, please replace each textarea
+      below with information about your proposed feature.
```

What do you think of condensing the intro sections in each template down into one paragraph without the heading? I think the additional headings make the template feel a little overwhelming.

```suggestion
- type: markdown
  attributes:
    value: |
      Thank you for contributing to Swift-DocC! Before you submit your issue, please replace each text area below with information about your proposed feature.
```

---

Got it and fixed